### PR TITLE
[#1790] Add documentation

### PR DESCRIPTION
**Pull request by sophokles73**
Added documentation of the MQTT adapter's experimental "custom message
mapping" feature to Device Registry Management API, Device Registration
API, MQTT adapter user and admin guides.

### CODE REVIEW

File: site/documentation/content/api/device-registration/index.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ In case of a successful invocation of the operation, the body of the response me
 | *device-id*      | *yes*     | *string*      | The ID of the device that is subject of the assertion. |
 | *via*            | *no*      | *array*       | The IDs (JSON strings) of gateways which may act on behalf of the device. This property MUST be set if any gateways are registered for the device. If the assertion request contained a *gateway_id* property and the response's *status* property has value `200` (indicating a successful assertion) then the array MUST at least contain the gateway ID from the request. |
 | *defaults*       | *no*      | *object*      | Default values to be used by protocol adapters for augmenting messages from devices with missing information like a *content type*. It is up to the discretion of a protocol adapter if and how to use the given default values when processing messages published by the device. |
+| *mapper*         | *no*      | *string*      | The (logical) name of a service that can be used to transform messages uploaded by the device before they are forwarded to downstream consumers. The client needs to map this name to the particular service to invoke. |
```
Comment (ctron) [inline site/documentation/content/api/device-registration/index.md:65]: I don't get the use of `(logical)` here. I think it can be dropped. Same below.

**Comment by sophokles73** [comment]
@ctron I have removed the *logical* word. Can you take another look?

File: site/documentation/content/api/device-registration/index.md (hunk 1 above)
Comment (ctron) [reply site/documentation/content/api/device-registration/index.md:65]: I think you still missed this one.

Comment (sophokles73) [reply site/documentation/content/api/device-registration/index.md:65]: damn ... I'll remove it ...

Comment (sophokles73) [reply site/documentation/content/api/device-registration/index.md]: ok, removed it, can you take a look again?

**Pull request merged**